plugins/woocommerce/includes/legacy/abstract-wc-legacy-order.php

```diff
@@ -329,7 +329,7 @@ public function get_product_from_item( $item ) {
 	/**
 	 * Set the customer address.
 	 * @param array $address Address data.
-	 * @param string $type billing or shipping.
+	 * @param string $type Type of address. Available options: 'billing' or 'shipping'.
```

Comment:
Is `available` the right word here? Maybe `supported` or `allowed` better captures what the method supports? Is it possible that an extension would add another address type? If so, this might be inaccurate.

Reply:
Maybe "natively supported"? I'm not aware that other extensions extend the options here, but it's a possibility -- if not now then in the future. 

(FWIW, I took the wording from one of the places where I saw it spelled out.)

Reply:
Could just take out the prefix to the values altogether:

```suggestion
	 * @param string $type Type of address. 'billing' or 'shipping'.
```

Reply:
> take out the prefix 

I'm going to go with that one. Thanks.